src/engines/kotor2/gui/chargen/charactergeneration.cpp

```diff
@@ -43,14 +43,16 @@ CharacterGeneration::CharacterGeneration(Module *module, CharacterGenerationInfo
 			getLabel("LBL_CLASS")->setText(TalkMan.getString(355));
 			break;
 		case kClassJediSentinel:
-			getLabel("LBL_CLASS")->setText(TalkMan.getString(356));
+			getLabel("LBL_CLASS")->setText(TalkMan.getString(353));
```

Comment:
Can you put that fix into the previous commit, as if that mistake never happened?